### PR TITLE
Dev/set author for song

### PR DESCRIPTION
**Pull request by tiborlakatoss**

### CODE REVIEW

File: projector-desktop/src/main/java/projector/controller/song/SongController.java [hunk 1]
```diff
@@ -13,30 +13,9 @@
...
-import javafx.scene.control.ListView;
-import javafx.scene.control.MenuItem;
-import javafx.scene.control.MultipleSelectionModel;
-import javafx.scene.control.SelectionMode;
-import javafx.scene.control.SingleSelectionModel;
-import javafx.scene.control.Slider;
-import javafx.scene.control.SplitPane;
-import javafx.scene.control.TextField;
-import javafx.scene.control.ToggleButton;
+import javafx.scene.control.*;
```
Comment (bence21) [inline projector-desktop/src/main/java/projector/controller/song/SongController.java]: Please revert the * imports

File: projector-desktop/src/main/java/projector/repository/ormLite/DatabaseHelper.java [hunk 2]
```diff
@@ -7,37 +7,18 @@
...
-import projector.model.Information;
-import projector.model.Language;
-import projector.model.Song;
-import projector.model.SongBook;
-import projector.model.SongBookSong;
-import projector.model.SongCollection;
-import projector.model.SongCollectionElement;
-import projector.model.SongVerse;
-import projector.model.VerseIndex;
+import projector.model.*;
```
Comment (bence21) [inline projector-desktop/src/main/java/projector/repository/ormLite/DatabaseHelper.java]: Also here

File: projector-desktop/src/main/resources/view/song/Song.fxml [hunk 3]
```diff
@@ -1,24 +1,11 @@
...
-<?import javafx.scene.control.Tooltip?>
-<?import javafx.scene.image.Image?>
-<?import javafx.scene.image.ImageView?>
-<?import javafx.scene.layout.BorderPane?>
-<?import javafx.scene.layout.GridPane?>
-<?import javafx.scene.layout.HBox?>
-<?import javafx.scene.layout.VBox?>
-<BorderPane xmlns:fx="http://javafx.com/fxml/1" prefHeight="560" prefWidth="800"
-            xmlns="http://javafx.com/javafx/8.0.121" fx:controller="projector.controller.song.SongController">
+<?import javafx.geometry.*?>
```
Comment (bence21) [inline projector-desktop/src/main/resources/view/song/Song.fxml]: also

File: projector-desktop/src/test/java/projector/controller/song/SongControllerTest.java [hunk 4]
```diff
@@ -73,6 +96,13 @@ public void clickNewSongButton() {
         deleteASong();
     }
 
+    @Test
+    public void checkAuthorTextField() {
+        clickOn("#searchTextField").write(il_iubesc_pe_el);
+        TextField authorTextField = find("#authorTextField");
+        Assert.assertEquals("Pinter Bela", authorTextField.getText());
```
Comment (bence21) [inline projector-desktop/src/test/java/projector/controller/song/SongControllerTest.java]: Pinter Bela is repeated

File: projector-desktop/src/main/resources/view/song/Song.fxml (hunk 3 above)
Comment (tiborlakatoss) [reply projector-desktop/src/main/resources/view/song/Song.fxml]: cannot import different classes. left it as it is.

**Pull request merged**